docs/taskruns.md

```diff
@@ -254,18 +254,23 @@ For more information, see the [`LimitRange` code example](../examples/v1beta1/ta
 
 ## Configuring the failure timeout
 
-You can use the `timeout` field to set the `TaskRun's` desired timeout value in minutes.
-If you do not specify this value in the `TaskRun`, the global default timeout value applies.
-If you set the timeout to 0, the `TaskRun` fails immediately upon encountering an error.
+You can use the `timeout` field to set the `TaskRun's` desired timeout value. If you do not specify this 
+value for the `TaskRun`, the global default timeout value applies. If you set the timeout to 0, the `TaskRun` will 
+have no timeout and will fail immediately upon encountering an error.
 
 The global default timeout is set to 60 minutes when you first install Tekton. You can set
 a different global default timeout value using the `default-timeout-minutes` field in
-[`config/config-defaults.yaml`](./../config/config-defaults.yaml).
+[`config/config-defaults.yaml`](./../config/config-defaults.yaml). If you set the global timeout to 0, 
+all `TaskRuns` that do not have a timeout set will have no timeout and will fail immediately upon 
+encountering an error.
 
 The `timeout` value is a `duration` conforming to Go's
 [`ParseDuration`](https://golang.org/pkg/time/#ParseDuration) format. For example, valid
-values are `1h30m`, `1h`, `1m`, and `60s`. If you set the global timeout to 0, all `TaskRuns`
-that do not have an individual timeout set will fail immediately upon encountering an error.
+values are `1h30m`, `1h`, `1m`, and `60s`. 
```

Comment:
is 0 not a valid value here? maybe worth mentioning that explicitly

Reply:
Yes, it is valid. I have added it in.